simulation/traffic_simulator/include/traffic_simulator/math/transfrom.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright 2015-2020 Tier IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TRAFFIC_SIMULATOR__MATH__TRANSFORM_HPP_
+#define TRAFFIC_SIMULATOR__MATH__TRANSFORM_HPP_
+
+#include <geometry_msgs/msg/pose.hpp>
+
+namespace traffic_simulator
+{
+namespace math
+{
+const geometry_msgs::msg::Pose getRelativePose(
+  const geometry_msgs::msg::Pose & from, const geometry_msgs::msg::Pose & to);
+}  // namespace math
+}  // namespace traffic_simulator
+
+#endif  // TRAFFIC_SIMULATOR__MATH__TRANSFORM_HPP_
```

Comment:
PLEASE fix "No newline at end of this".

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/372/commits/e408a9a5425d1c1e84f9a926594993abee52f75b